src/routes/intro/IntroScreen.tsx

```diff
@@ -0,0 +1,83 @@
+import { useFocusEffect } from '@react-navigation/native'
+import { StackNavigationProp } from '@react-navigation/stack'
+import React from 'react'
+import styled from 'styled-components/native'
+
+import Button from '../../components/Button'
+import DisciplineListItem from '../../components/DisciplineListItem'
+import Header from '../../components/Header'
+import ServerResponseHandler from '../../components/ServerResponseHandler'
+import { ContentSecondary } from '../../components/text/Content'
+import { BUTTONS_THEME } from '../../constants/data'
+import { Discipline } from '../../constants/endpoints'
+import labels from '../../constants/labels.json'
+import { useLoadDisciplines } from '../../hooks/useLoadDisciplines'
+import useReadSelectedProfessions from '../../hooks/useReadSelectedProfessions'
+import { RoutesParams } from '../../navigation/NavigationTypes'
+
+const Root = styled.ScrollView`
+  background-color: ${props => props.theme.colors.background};
+  height: 100%;
+`
+
+const ButtonContainer = styled.View`
+  margin: ${props => props.theme.spacings.md} auto;
+`
+
+const StyledText = styled(ContentSecondary)`
+  margin-top: ${props => props.theme.spacings.xxl};
+  text-align: center;
+  margin-bottom: ${props => props.theme.spacings.lg};
+`
+
+interface IntroScreenProps {
+  navigation: StackNavigationProp<RoutesParams, 'Intro'>
+}
+
+const IntroScreen = ({ navigation }: IntroScreenProps): JSX.Element => {
+  const { data: disciplines, error, loading, refresh } = useLoadDisciplines(null)
+  const { data: selectedProfessions, refresh: refreshSelectedProfessions } = useReadSelectedProfessions()
```

Comment:
Why don't you handle error and loading the same way as those from `useLoadDiscipline`?

Reply:
If `useLoadDiscipline` is still loading, we have no data to display, whereas if the `useReadSelectedProfession` is still loading, we can display the data, but only the green-check-icon is missing. Not sure if the reading from AsyncStorage can fail :thinking: At least it is far less common than a failing api-request.

If you want I can open a ticket, where we implement an appropriate error handling.

Reply:
No, sounds good to me :+1: